src/limit-order/predicate-factory.ts

```diff
@@ -1,5 +1,5 @@
-import {BigNumber} from '@ethersproject/bignumber'
 import {add0x, toBN} from '../utils'
+import {BN} from '../utils/bytes/bn'
 
 export class PredicateFactory {
```

Comment:
I don't think we still need this. Because deadline and nonce you can set in maker traits